src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliCommand.cs

```diff
@@ -196,5 +196,27 @@ private static string GetMandatoryMsBuildSettings(string buildConfiguration)
 
             return $"{NoMsBuildZombieProcesses} {EnforceOptimizations}";
         }
+
+        private static string BuildAddPackageCommand(NuGetReference reference)
+        {
+            var commadBuilder = new StringBuilder();
+            commadBuilder.Append("add package ");
+            commadBuilder.Append(reference.PackageName);
+            if (string.IsNullOrWhiteSpace(reference.PackageVersion) == false)
+            {
+                commadBuilder.Append(" -v ");
+                commadBuilder.Append(reference.PackageVersion);
+            }
+            if (reference.PackageSource != null)
+            {
+                commadBuilder.Append(" -s ");
+                commadBuilder.Append(reference.PackageSource);
+            }
+            if (reference.Prerelease)
+            {
+                commadBuilder.Append(" --prerelease");
+            }
+            return commadBuilder.ToString();
+        }
```

Comment:
nits:
* typo => commadBuilder => comma**n**dBuilder 
* `boolean == false` we are rather using `!boolean` (style consistency with the rest of the repo)

```suggestion
            var commandBuilder = new StringBuilder();
            commadBuilder.Append("add package ");
            commadBuilder.Append(reference.PackageName);
            if (!string.IsNullOrWhiteSpace(reference.PackageVersion))
            {
                commadBuilder.Append(" -v ");
                commadBuilder.Append(reference.PackageVersion);
            }
            if (reference.PackageSource != null)
            {
                commadBuilder.Append(" -s ");
                commadBuilder.Append(reference.PackageSource);
            }
            if (reference.Prerelease)
            {
                commadBuilder.Append(" --prerelease");
            }
            return commadBuilder.ToString();
        }
```